tests/addresslib/parser_mailbox_test.py

```diff
@@ -572,17 +555,6 @@ def test_local_part():
 def test_domain():
     "Grammar: domain -> dot-atom"
 
-    # test length limits
-    max_domain_len = ''.join(['a'*62, '.', 'b'*62, '.', 'c'*63, '.', 'd'*63])
-    overlimit_domain_one = ''.join(['a'*62, '.', 'b'*62, '.', 'c'*63, '.', 'd'*64])
-    overlimit_domain_two = ''.join(['a'*62, '.', 'b'*62, '.', 'c'*63, '.', 'd'*63, '.', 'a'])
-    run_mailbox_test(''.join(['b@', 'a'*63]), ''.join(['b@', 'a'*63]))
-    run_mailbox_test(''.join(['b@', 'a'*64]), None)
-    run_mailbox_test(''.join(['a@', max_domain_len]), ''.join(['a@', max_domain_len]))
-    run_mailbox_test(''.join(['a@', overlimit_domain_one]), None)
-    run_mailbox_test(''.join(['a@', overlimit_domain_two]), None)
```

Comment:
These tests mostly dealt with length limits on the atoms that made up dot-atoms in domains. All of this was outside of the RFC. There are checks for domains that exceed `MAX_ADDRESS_LENGTH` elsewhere.